11-payment-encoding.md

```diff
@@ -140,6 +141,9 @@ Currently defined tagged fields are:
    * `fee_base_msat` (32 bits, big-endian)
    * `fee_proportional_millionths` (32 bits, big-endian)
    * `cltv_expiry_delta` (16 bits, big-endian)
+* `9` (5): `data_length` variable. One or more bytes containing features
```

Comment:
Any particular reason why `9` is chosen here when the existing defined tags are letters?

Reply:
I think `9` was chosen to refer to Bolt 9 (which is where feature bits are usually defined - even though this might create confusion between node feature bits and invoice feature bits).

Reply:
No, it seemed the least desirable "real estate", character-wise, and since it doesn't have a specific meaning, I thought I'd use it. `f` was already taken, of course.